ambry-clustermap/src/test/java/com.github.ambry.clustermap/ClusterMapManagerTest.java

```diff
@@ -137,7 +149,7 @@ public void addNewPartition()
   }
 
   @Test
-  public void bestEffortAllocation()
+  public void nonRackAwareAllocationTest()
```

Comment:
Because the hardware layout does not have rack info in this test case. I'm open to suggestions for better names though.